suite-native/module-accounts/src/components/TransactionListHeader.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { memo } from 'react';
+import { useSelector } from 'react-redux';
+
+import { useAtom } from 'jotai';
+
+import { AccountBalance } from '@suite-native/accounts';
+import { Box, Button, Divider, Text } from '@suite-native/atoms';
+import { AccountKey } from '@suite-common/suite-types';
+import { AccountsRootState, selectAccountByKey } from '@suite-common/wallet-core';
+import { isSendReceiveActionsVisibleAtom } from '@suite-native/navigation';
+
+import { AccountDetailGraph } from './AccountDetailGraph';
+
+type AccountDetailHeaderProps = {
+    accountKey: AccountKey;
+};
+
+export const TransactionListHeader = memo(({ accountKey }: AccountDetailHeaderProps) => {
+    const [_, setIsSendReceiveActionsVisible] = useAtom(isSendReceiveActionsVisibleAtom);
+    const account = useSelector((state: AccountsRootState) =>
+        selectAccountByKey(state, accountKey),
+    );
+
+    const accountHasTransactions = !!account?.history.total ?? false;
```

Comment:
this should be moved to selector because it will be useful in more places than just here for sure

Reply:
yeah, good point. Used selector